### PR TITLE
Isolated removal

### PR DESCRIPTION
**Pull request by jbradberry**
##### SUMMARY
Removal of the isolated nodes feature.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - API

##### AWX VERSION
```

```

### CODE REVIEW

File: awx/main/management/commands/register_queue.py [hunk 1]
```diff
@@ -95,11 +80,6 @@ def register(self):
                 elif not created:
                     print("Instance Group already registered {}".format(ig.name))
 
-                if self.controller:
-                    (ig_ctrl, changed2) = self.update_instance_group_controller(ig)
```
Comment (AlanCoding) [inline awx/main/management/commands/register_queue.py:99]: You should get rid of the other uses of `changed2` here. It should work as it is, but we can get rid of the now-useless variable.

Comment (jbradberry) [reply awx/main/management/commands/register_queue.py:99]: This is all very much a work in progress.  Reviews are premature.

File: awx/main/models/jobs.py [hunk 2]
```diff
@@ -759,7 +755,7 @@ def display_artifacts(self):
 
     @property
     def can_run_containerized(self):
-        return any([ig for ig in self.preferred_instance_groups if ig.is_container_group])
+        return True
```
Comment (AlanCoding) [inline awx/main/models/jobs.py:758]: Calling out a notable change here. I agree with it, but will make sure to follow the use of the property to double-check.

Comment (AlanCoding) [reply awx/main/models/jobs.py:758]: Finished looking, I feel pretty confident in this.

Comment (jbradberry) [reply awx/main/models/jobs.py:758]: Grep shows 5 instances:

```
./awx/main/scheduler/task_manager.py:282:                elif task.can_run_cont>
./awx/main/scheduler/task_manager.py:485:                if task.can_run_contai>
./awx/main/models/jobs.py:757:    def can_run_containerized(self):
./awx/main/models/ad_hoc_commands.py:154:    def can_run_containerized(self):
./awx/main/models/unified_jobs.py:741:    def can_run_containerized(self):
```

3 are the defs, 1 is the one I know I moved this `any()` check to.  The other one (`./awx/main/scheduler/task_manager.py:485`) seems to be doing this same `.is_container_group` logic in a more complex loop.

**Review by AlanCoding — APPROVED** [review]

**Review by elyezer — APPROVED** [review]

**Review by beeankha — APPROVED** [review]

**Pull request merged**